src/functions/Coverage.ps1

```diff
@@ -1030,6 +1029,224 @@ function Get-JaCoCoReportXml {
     return $xml
 }
 
+function Get-CoberturaReportXml {
+    param (
+        [parameter(Mandatory = $true)]
+        [object] $CoverageReport,
+        [parameter(Mandatory = $true)]
+        [long] $TotalMilliseconds
+    )
+
+    if ($null -eq $CoverageReport -or $CoverageReport.NumberOfCommandsAnalyzed -eq 0) {
+        return [string]::Empty
+    }
+
+    # Report uses unix epoch time format (milliseconds since midnight 1/1/1970 UTC)
+    [long] $endTime = [System.DateTimeOffset]::UtcNow.ToUnixTimeMilliseconds()
+    [long] $startTime = [math]::Floor($endTime - $TotalMilliseconds)
```

Comment:
Source for epoch time format: https://github.com/cobertura/cobertura/blob/0ff963284cecaace30f409a977dccb07c41a5a8f/cobertura/src/main/java/net/sourceforge/cobertura/reporting/xml/XMLReport.java#L89